botcore/__init__.py

```diff
@@ -1,11 +1,15 @@
 """Useful utilities and tools for Discord bot development."""
 
-from botcore import exts, site_api, utils
+from botcore import async_stats, exts, site_api, utils
+from botcore._bot import BotBase, StartupError
 
 __all__ = [
+    async_stats,
+    BotBase,
     exts,
     utils,
     site_api,
```

Comment:
I suppose this is out of scope, but same thing I said for `async_stats` applies here. Either export just `ResponseCodeError` and `APIClient`, or add an `__all__` to it.

Reply:
https://github.com/python-discord/bot-core/pull/42/commits/7d67c3fd6ebabe9dc92ed379e7705e36cf147c5d